### PR TITLE
Interactive-list

### PR DESCRIPTION
**Pull request by ethred**
Add a new JavaScript file checked.js and import it as a module
Add event listener to the checkbox (change)
Update items object's value for completed key upon user actions. and update the local storage
Implement a function for the "Clear all completed" button (use filter() method).
Save every update to Local Storage

### CODE REVIEW

**Review by usorfaitheloho — COMMENTED** [review]
Hi @ethred , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an empty README file. Kindly add A README file having the microverse template and resubmit your project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by ethred** [comment]
hi @usorfaitheloho  thank you for you feedback i update the readme file

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @ethred 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅
 - [x] You have written a clean code ✅
 - [x]  Most of the requirement has been meet. ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @ethred, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/checked.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    toDelete = () => {
+      todos = todos.filter((todo) => todo.completed !== true);
+      todos = todos.map((todo, index) => ({
+        ...todo,
+        index: index + 1,
+      }));
+      datas.setodos(todos);
+      window.location.reload();
```
Comment (AliRezaBmeDu) [inline src/modules/checked.js:23]: You may consider writing separate functions for `checkToDo `and `toDelete` instead of using `class`

**Pull request merged**